public/next-assets/document.svg

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
Instead of using svg preferabally convert svg to jsx component and import it directly so it will avoid extra network call + this icon has very much similarity wil old google docs icon, which can be misleading so avoid that as well